msal/src/main/java/com/microsoft/identity/client/Account.java

```diff
@@ -146,4 +147,17 @@ public String getUsername() {
 
         return MISSING_FROM_THE_TOKEN_RESPONSE;
     }
+
+    @Override
+    @NonNull
+    public String getAuthority() {
+        if (null != getClaims()) {
```

Comment:
nit: please add TODO here.

Reply:
Why?

Are you referring to the case of B2C?

Reply:
yes. we should add TODO for any in-progress/partially finished work.

Reply:
Addressed